components/template/examples/basic/butterfly.template

```diff
@@ -1,19 +1,18 @@
 <a-mixin id="wing" geometry="primitive: box; depth: .005; height: 1; width: 0.5"
                    pivot="0.15 0" material="color: red" scale="0.5 0.625 0.5"></a-mixin>
-<a-mixin id="flap" attribute="rotation" direction="alternate" repeat="indefinite"></a-mixin>
```

Comment:
It does not appear to be possible to use mixins in this way with the animation component, so I have duplicated the properties on each wing.